app/services/request.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class Request
+  def initialize(connection)
+    @connection = connection
+  end
+
+  def retrieve(path)
+    response, status = get_json(path)
+    status == 200 ? response : errors(response)
```

Comment:
Going forward it might be worth creating an api error/exceptions class to handle any non 200 responses. Worth doing when the app gets bigger and we set up any api monitoring etc